docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -183,7 +183,7 @@ sdlManager.getSystemCapabilityManager().addOnSystemCapabilityListener(SDL.rpc.en
 !@
 
 #### Getting Module Data Location and Service Areas (RPC v6.0+)
-With the saved remote control capabilities struct you can build a UI to display modules to the user by getting the location of the module and the area that it services. This will map to the grid you receive in **Setting the User's Seat** below.
+With the saved remote control capabilities struct you can get the location of the each module and the area that it services. This will map to the `grid` [graphic](#the-grid) below.  This information is useful for creating a custom UI. 
```

Comment:
```suggestion
With the saved remote control capabilities struct you can get the location of the each module and the area that it services. This will map to the `grid` [graphic](#the-grid) below. This information is useful for creating a custom UI.
```